hamocc/mo_ncout_hamocc.F90

```diff
@@ -1770,6 +1858,24 @@ subroutine hamoccvardef(iogrp,timeunits,calendar,cmpflg)
       call ncdefvar3d(SRF_ATMBROMO(iogrp),cmpflg,'p',                           &
            &   'atmbromo','Atmospheric bromoform',' ','ppt',0)
     endif
+    if (use_dom) then
+      call ncdefvar3d(INT_EXUDL(iogrp),cmpflg,'p','exudlint',                   &
+           &   'Integrated phy exudation to DOC-L',' ','mol C m-2 s-1',0)
+      call ncdefvar3d(INT_EXUDSL(iogrp),cmpflg,'p','exudslint',                 &
+           &   'Integrated phy exudation to DOC-SL',' ','mol C m-2 s-1',0)
+      call ncdefvar3d(INT_EXCRL(iogrp),cmpflg,'p','excrlint',                   &
+           &   'Integrated zoo excretion to DOC-SL',' ','mol C m-2 s-1',0)
```

Comment:
@tjiputra: Should this read DOC-L (l. 1867)

Reply:
yes. thank you for noticing this.